javascript-nodejs-stream/receive.js

```diff
@@ -0,0 +1,36 @@
+const rabbit = require("rabbitmq-stream-js-client")
+
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms))
+
+async function main() {
+    const streamName = "test-queue-stream"
+
+    console.log("Connecting...");
+    const client = await rabbit.connect({
+        hostname: "localhost",
+        port: 5552,
+        username: "rabbit",
+        password: "rabbit",
+        vhost: "/",
+    })
+
+    console.log("Making sure the stream exists...");
+    await client.createStream({ stream: streamName, arguments: {} })
```

Comment:
Can you please add the size retention https://github.com/rabbitmq/rabbitmq-tutorials/pull/383/files#diff-aaa61986b4ca5dbb6e120b1304f70f4014885275501a001306d4879b5389b017R30 ?  like 2 GB ? Thank you